app/assets/javascripts/miq_application.js

```diff
@@ -790,15 +790,20 @@ function miqAjaxButtonSend(url, serialize_fields) {
 
 // Function to generate an Ajax request
 function miqAjax(url, serialize_fields) {
-  var data = undefined;
+  var data = 'undefined';
```

Comment:
This is broken, you really need to initialize to `undefined`, not `'undefined'` otherwise if you don't overwrite it, it will send malformed data to the server - sends a string instead of nothing.